examples/asr/experimental/wav2vec/configs/wav2vec_pretrain.yaml

```diff
@@ -0,0 +1,122 @@
+name: &name "Wav2vec_Pretrain"
+
+model:
+  wav2vec:
+    loss:
+      prob_ppl_weight: 0.1
```

Comment:
Do you think it is a good idea to copy the some of the descriptions for each config here so that user may get more familiar with these configs. We can also add some comment on the top of this file and direct them to the config file for more detail. Still maybe not bad to have some comments here. What do you think?